test/reducer.test.js

```diff
@@ -1,12 +1,34 @@
 /* eslint flowtype/require-parameter-type: 0 */
-import reducer, { initialState, networkSelector } from '../src/redux/reducer';
+import { isEqual } from 'lodash';
+import createReducer, {
+  initialState,
+  networkSelector,
+} from '../src/redux/createReducer';
 import * as actionCreators from '../src/redux/actionCreators';
+import getSimilarActionInQueue from '../src/utils/getSimilarActionInQueue';
+
+const networkReducer = createReducer();
 
 const getState = (isConnected = false, ...actionQueue) => ({
   isConnected,
   actionQueue,
 });
 
+function comparisonFn(action, actionQueue) {
```

Comment:
Since you are using the `comparisonFn` way down, I'd colocate it at the beginning of the `describe` block for better placement (Line 135)